action-server/Makefile

```diff
@@ -60,6 +60,4 @@ format:
 test:
 	${ACTIVATE_VENV} py.test "${TESTS_FOLDER}/${TEST_FILE}" \
 		--cov-report term-missing:skip-covered \
-		--cov-report html \
-		--cov-fail-under=80 \
-		--cov ${SOURCE_FOLDER}
```

Comment:
Why is that no longer necessary?

Reply:
those are now included in the `setup.cfg` file